AzureMonitorAgent/agent.py

```diff
@@ -327,6 +327,14 @@ def install():
     # Copy the PA and agentlauncher binaries
     copy_pa_binaries()
 
+    # Retry install for aarch64 rhel8 VMs as initial install fails to create symlink to /etc/systemd/system/azuremonitoragent.service
+    # in /etc/systemd/system/multi-user.target.wants/azuremonitoragent.service
+    if vm_dist.replace(' ','').lower().startswith('redhat') and vm_ver == '8.6' and platform.machine() == 'aarch64':
```

Comment:
Use [lstrip](https://python-reference.readthedocs.io/en/latest/docs/str/lstrip.html) instead of .replace(' ','')